From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ruhan <35257374+RuhanSA079@users.noreply.github.com>
Date: Wed, 2 Sep 2026 00:00:00 +0000
Subject: [PATCH] radxa-cm3-io: use the standard 115200 console baudrate

The Radxa CM3 IO defconfig and its device tree default the debug/console
UART to a non-standard 1500000 baud. Switch it to the standard 115200
so the U-Boot console matches common serial adapters and the rest of
the boards in this repo.

---
 arch/arm/dts/rk3566-radxa-cm3-io.dts | 2 +-
 configs/radxa-cm3-io-rk3566_defconfig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/rk3566-radxa-cm3-io.dts b/arch/arm/dts/rk3566-radxa-cm3-io.dts
--- a/arch/arm/dts/rk3566-radxa-cm3-io.dts
+++ b/arch/arm/dts/rk3566-radxa-cm3-io.dts
@@ -18,7 +18,7 @@
 	};

 	chosen: chosen {
-		stdout-path = "serial2:1500000n8";
+		stdout-path = "serial2:115200n8";
 	};

 	gmac1_clkin: external-gmac1-clock {
diff --git a/configs/radxa-cm3-io-rk3566_defconfig b/configs/radxa-cm3-io-rk3566_defconfig
--- a/configs/radxa-cm3-io-rk3566_defconfig
+++ b/configs/radxa-cm3-io-rk3566_defconfig
@@ -71,7 +71,7 @@
 CONFIG_REGULATOR_RK8XX=y
 CONFIG_PWM_ROCKCHIP=y
 CONFIG_SPL_RAM=y
-CONFIG_BAUDRATE=1500000
+CONFIG_BAUDRATE=115200
 CONFIG_DEBUG_UART_SHIFT=2
 CONFIG_SYS_NS16550_MEM32=y
 CONFIG_SYSRESET=y
--
2.43.0
